src/app/detail/ui/Sidebar.tsx

```diff
@@ -117,7 +119,7 @@ export const Sidebar = ({
               updateStatusValue(value)
               updateWorkflowState(value)
             }}
-            disabled={disabled}
+            disabled={workflowDisabled}
```

Comment:
Is there a reason why we are completely removing `disabled` and replacing it with `workflowDisabled` here?

Reply:
disabled prop is being passed from page.tsx as true for clients, since this ticket requires clients to be able to edit workflowstates, i introduced workflowDisabled prop (default false) so that it could be changed according to need in the future.